fecgov/openFEC
# Remove unused filters from Swagger API docs

## Summary (required)

Missed part of issue that should have been addressed with this issue https://github.com/fecgov/openFEC/issues/3504

Remove unused MUR filters from Swagger API docs

## How to test the changes locally

Go to http://localhost:5000/developers/#/legal/get_legal_search_ and make sure there are no `mur_` filters

## Impacted areas of the application
List general components of the application that this PR will affect:

-  Swagger docs for legal



## Related PRs
List related PRs against other branches:

https://github.com/fecgov/openFEC/issues/3504
